README.md

```diff
@@ -8,7 +8,41 @@ A library to **sub**mit e**xt**rinsics to a [substrate](https://github.com/parit
 
 ## Usage
 
-See [examples](./examples).
+### Downloading metadata from a Substrate node
+
+Use the `subxt-cli` CLI tool to download the metadata for your target runtime from a Substrate node.
```

Comment:
I guess it could be any type of node yes? Polkadot, kusama, <your parachain here>?

```suggestion
Use the `subxt-cli` CLI tool to download the metadata for your target runtime from a node.
```

Reply:
Indeed, I suppose I meant to say any node built *using* substrate